ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -121,13 +121,24 @@ post_provision_config_nodes() {
     rm -f /etc/profile.d/openmpi.sh
     rm -f /tmp/daos_control.log
     dnf -y install redhat-lsb-core
+
+    # force install of avocado 82.0
+    dnf -y erase avocado{,-common} python2-avocado{,-plugins-{output-html,varianter-yaml-to-mux}} python36-PyYAML
+    pip3 install --upgrade pip
+    pip3 install "avocado-framework<83.0"
+    pip3 install "avocado-framework-plugin-result-html<83.0"
+    pip3 install "avocado-framework-plugin-varianter-yaml-to-mux<83.0"
+    pip3 install clustershell
+
```

Comment:
Avocado 69.2 LTS is in EPEL7 now and being installed on the CI nodes.

Is there anything specific about >69.2 <83.0 that is needed?

Reply:
Any comment here @mjean308?